guides/common/modules/proc_assigning-content-view-environments-to-hosts-by-using-subscription-manager.adoc

```diff
@@ -0,0 +1,38 @@
+[id="assigning-content-view-environments-to-hosts-by-using-subscription-manager"]
+= Assigning content view environments to hosts by using Subscription Manager
+
+You can assign content view environments to hosts by using Subscription Manager.
+
+.CLI procedure
+. Display the available environments and identify which ones to pass in:
++
+[options="nowrap" subs="+quotes"]
+----
+# subscription-manager environments --list
+----
+. Set the content view environments for the host using `subscription-manager`:
++
+[options="nowrap" subs="+quotes"]
+----
+# subscription-manager environments --set Library/cv1,Library
```

Comment:
Is this correct? Doesn't it always need to be in the `<lce>/<cv>` format?

Reply:
`Library` is the only exception to that. It's special ;)

Reply:
Ok, gotcha, I guess this part relates to it
```
= Content view environment categories

The content view environments available in your organization fall into the following categories:

Library environment::
{Project} assigns the Library environment to represent the Library lifecycle environment and the *Default Organization View* content view.
```

Reply:
Yes exactly! :100: